src/command_modules/azure-cli-storage/azure/cli/command_modules/storage/commands.py

```diff
@@ -44,21 +47,26 @@ def _dont_fail_not_exist(ex):
         raise ex
 
 
+transform_keys_result = lambda x: x.keys if hasattr(x, 'keys') else x  # noqa: E731 lambda vs def
```

Comment:
@troydai  I could have done this inline as was the case before but since this method is slightly longer, I assigned the lambda to a variable.

I'm happy to move it to a normal method or back to inline. Thoughts?